charts/handbrake/0.0.1/test_values.yaml

```diff
@@ -76,15 +51,71 @@ persistence:
     mountPath: /output
 
 ## Somehow must enable user to add as many watch folders
-## All extra folders must start with "watch" and an index after
+## All folders must start with "watch" and an index after
 ## eg. /watch1, /watch2
 ## This is importand because settings can overrode to each watch folder exclusivly, with env vars with _# as a suffix
 ## eg. AUTOMATED_CONVERSION_PRESET_1 
 
+# Path to your optical drive device in the container
+additionalVolumeMounts: []
+#  - name: opticaldrive
+#    mountPath: /dev/serial/by-id/sr0-
+
+# Path to your zwave device on the host
+additionalVolumes: []
+#  - name: opticaldrive
+#    hostPath:
+#      path: /dev/serial/by-id/sr0-
+
+# affinity:
+#   nodeAffinity:
+#     requiredDuringSchedulingIgnoredDuringExecution:
+#       nodeSelectorTerms:
+#       - matchExpressions:
+#         - key: app
+#           operator: In
+#           values:
+#           - opticaldrive-controller
+
+#appVolumeMounts:
+#  config:
+#    enabled: true
+#    emptyDir: false
+#    datasetName: "config"
+#    mountPath: "/config"
+#    hostPathEnabled: false
+#  storage:
+#    enabled: true
+#    emptyDir: false
+#    datasetName: "storage"
+#    mountPath: "/storage"
+#    hostPathEnabled: false
+#  watch:
+#    enabled: true
+#    emptyDir: false
+#    datasetName: "watch"
+#    mountPath: "/watch"
+#    hostPathEnabled: false
+#  output:
+#    enabled: true
+#    emptyDir: false
+#    datasetName: "output"
+#    mountPath: "/output"
+#    hostPathEnabled: false
```

Comment:
Needs to be enabled on test_values.
And setPermissions: true need to be added to all of them.